wagtailsharing/tests/settings.py

```diff
@@ -27,7 +29,9 @@
 
 WAGTAIL_APPS = (
     "wagtail.contrib.forms",
-    "wagtail.contrib.modeladmin",
+    "wagtail.contrib.modeladmin"
+    if WAGTAIL_VERSION <= (5, 0)
```

Comment:
the Wagtail convention is to check if greater than a given version if you have branches. So this would be:

```python
"wagtail.snippets"
if WAGTAIL_VERSION >= (5, 0)
else "wagtail.contrib.modeladmin"
```

Reply:
Shouldn't that be
`if WAGTAIL_VERSION >= (5, 1)`, since it's specifically a Wagtail 5.1 upgrade consideration?
Thanks